src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataInternalLatestIT.java

```diff
@@ -143,7 +143,7 @@ public void testGetFunctionColumns() throws SQLException {
     assertFalse(resultSet.next());
     resultSet = databaseMetaData.getFunctionColumns(null, "%", "%", "%");
     // we have 81 columns returned
-    assertEquals(81, getSizeOfResultSet(resultSet));
+    assertEquals(4, getSizeOfResultSet(resultSet));
```

Comment:
Why does the comment still say `81` if the new value is `4`? And other than knowing that the query returns rows, does it really matter whether it returns 4 or 81? Isn't this test really just making sure we populate the result set with _something_?